src/test/components/MenuButtons.test.js

```diff
@@ -361,26 +358,39 @@ describe('app/MenuButtons', function() {
   });
 
   describe('<MetaInfoPanel>', function() {
-    async function setup(profile: Profile) {
+    async function setupForMetaInfoPanel(profile: Profile) {
       jest
         .spyOn(Date.prototype, 'toLocaleString')
         .mockImplementation(function() {
           // eslint-disable-next-line babel/no-invalid-this
           return 'toLocaleString ' + this.toUTCString();
         });
 
+      jest.useFakeTimers();
+
       const store = blankStore();
 
       // Note that we dispatch this action directly instead of using viewProfile
       // or loadProfile because we want to control tightly how symbolication is
       // started in these tests.
       await store.dispatch(loadProfile(profile, { skipSymbolication: true }));
 
-      return render(
-        <Provider store={store}>
-          <MetaInfoPanel />
-        </Provider>
-      );
```

Comment:
Now, instead of rendering the panel directly, we render the full `<MenuButtons>` component and click on the button. The reason is that in the next patch the panel will sometimes have more than this component, and so this is closer to what a user would do and see.